app/src/processing/app/contrib/ContributionTab.java

```diff
@@ -92,10 +89,9 @@ public boolean matches(Contribution contrib) {
       }
     };
 
-    contribListing = ContributionListing.getInstance();
     statusPanel = new StatusPanel(this, 650);
     contributionListPanel = new ListPanel(this, filter);
-    contribListing.addListener(contributionListPanel);
+    ManagerFrame.contributionListing.addListener(contributionListPanel);
```

Comment:
This excessive de-referencing is bad form. We're not going to trade a single declaration for static access to a double-dereferenced method, requiring even more typing.